docs/usage/configuration/config_documentation.md

```diff
@@ -2837,6 +2837,17 @@ Example configuration:
 ```yaml
 track_appservice_user_ips: true
 ```
+---
+### `use_appservice_legacy_authorization`
+
+Whether to send the application service access token via the `access_token` query parameter.
+Defaults to false. This option is considered insecure and is not recommended. 
+
+Example configuration:
+```yaml
+"use_appservice_legacy_authorization": True 
```

Comment:
```suggestion
use_appservice_legacy_authorization: True 
```

We seem to use `true` above instead of `True`. Not sure how consistent we are in this doc though.

Reply:
What's above is probably correct, I've changed it here and in the upgrade doc.